### PR TITLE
GL translation

### PR DESCRIPTION
**Pull request by xurxodiz**
Let me preface by stating that this is a serious job that should have been comissioned from a **paid professional** translator—but _until_ that time comes, this fixes a glaring omission and obligation by the government to provide service to all citizens in their official language of choosing.

Review & feedback are appreciated.

### CODE REVIEW

File: app/src/main/res/values-gl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+        funcionar correctamente.</string>
+    <string name="dp3t_sdk_service_notification_error_battery_optimization"><b>Aforro de enerxía activado</b>\nDebes desactivalo para o Radar COVID funcionar correctamente.</string>
+    <string name="dp3t_sdk_service_notification_error_bluetooth_not_supported">O Bluetooth de baixa enerxía non é compatíbel con este dispositivo.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_server">Erro ao cargar novos datos de infección.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_network">Erro ao cargar novos datos de infección.</string>
+    <string name="dp3t_sdk_service_notification_error_no_space">Erro ao cargar novos datos de infección. Non hai espazo dispoñíbel neste dispositivo.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_ssltls">Erro ao cargar novos datos de infección.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_timing">É preciso configurares en Axustes aa actualización automática de data e hora.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_signature">Error ao cargar novos datos de infección.</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:345]:
```suggestion
    <string name="dp3t_sdk_service_notification_error_sync_signature">Erro ao cargar novos datos de infección.</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="dp3t_sdk_service_notification_error_sync_api">Erro ao cargar nuevos datos de infección.</string>
+    <string name="dp3t_sdk_service_notification_error_gaen_not_available">As notificaciones de exposición COVID-19 non están dispoñíbeis neste dispositivo.</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:347]:
```suggestion
    <string name="dp3t_sdk_service_notification_error_gaen_not_available">As notificacións de exposición COVID-19 non están dispoñíbeis neste dispositivo.</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="dp3t_sdk_service_notification_error_bluetooth_disabled">Fai o favor de activar o Bluetooth. É imprescindíbel para &appname;
+        funcionar correctamente.</string>
+    <string name="dp3t_sdk_service_notification_error_battery_optimization"><b>Aforro de enerxía activado</b>\nDebes desactivalo para o Radar COVID funcionar correctamente.</string>
+    <string name="dp3t_sdk_service_notification_error_bluetooth_not_supported">O Bluetooth de baixa enerxía non é compatíbel con este dispositivo.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_server">Erro ao cargar novos datos de infección.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_network">Erro ao cargar novos datos de infección.</string>
+    <string name="dp3t_sdk_service_notification_error_no_space">Erro ao cargar novos datos de infección. Non hai espazo dispoñíbel neste dispositivo.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_ssltls">Erro ao cargar novos datos de infección.</string>
+    <string name="dp3t_sdk_service_notification_error_sync_timing">É preciso configurares en Axustes aa actualización automática de data e hora.</string>
```
Comment (xurxodiz) [inline app/src/main/res/values-gl/strings.xml:344]:
```suggestion
    <string name="dp3t_sdk_service_notification_error_sync_timing">É preciso configurares en Axustes a actualización automática de data e hora.</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="privacy_p12_title"><b>12. Política de cookies</b></string>
+    <string name="privacy_p12_content">Utilizamos só cookies técnicas que permiten ao usuario a navegación e a utilización das diferentes opcións ou servizos que se ofrecen na Aplicación como, por exemplo, acceder a partes de acceso restrinxido ou utilizar elementos de seguridade durante a navegación.\n\n\nLin o documento POLÍTICA DE PRIVACIDADE DA APLICACIÓN “Radar COVID”.</string>
+
+    /********** Onboarding ************/
+    <string name="onboarding_title_step_1">Benvido a &appname;</string>
+    <string name="onboarding_subtitle_step_1">Para te dar o mellor servizo, fai o favor de seleccionar o teu idioma. Se non é o castelán, reinicia <b>Radar COVID</b> despois de o seleccionar</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:94]:
```suggestion
    <string name="onboarding_subtitle_step_1">Para dármosche o mellor servizo, por favor, selecciona o teu idioma. Se non é o castelán, reinicia <b>Radar COVID</b> despois de o seleccionar</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+
+    /********** Home ************/
+    <string name="exposition_block_low_title">Exposición baixa</string>
+    <string name="exposition_block_medium_title">Exposición media</string>
+    <string name="exposition_block_high_title">Contacto estreito</string>
+    <string name="exposition_block_infected_title">COVID-19 Positivo</string>
+
+    <string name="exposition_block_low_description"><b>Informarémoste no caso dun posíbel contacto de risco.</b></string>
+    <string name="exposition_block_high_description"><b>Estiveches en contacto cunha persoa contaxiada de COVID-19.\nInforma no 900 112 061 (de balde)</b>.</string>
+    <string name="exposition_block_infected_description"><b>O teu diagnóstico foi enviado.\nFai o favor de aillarte durante 14 días</b>.</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:158]:
```suggestion
    <string name="exposition_block_infected_description"><b>O teu diagnóstico foi enviado.\nPor favor, aíllate durante 14 días</b>.</string>
```

**Comment by nosolosw** [comment]
:wave: @xurxodiz estaba a enviar suxestións de cambio, pero quizás che facilito a integración se fago un comentario xeral de actualizar "fai o favor" pola forma "por favor" ou similar dependendo do contexto. Así podes facer ti un reemplazo rápido.

File: app/src/main/res/values-gl/strings.xml [hunk 6]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="exposure_detail_low_symptoms_what_to_do"><u>Como actuar se teño síntomas ></u></string>
+    <string name="exposure_detail_low_symptoms_what_to_do_url">https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/documentos/20200325_Decalogo_como_actuar_COVID19.pdf</string>
+
+    /********** Exposure detail high ************/
+    <string name="exposure_detail_high_title_small">RISCO ALTO</string>
+    <string name="exposure_detail_high_title">A túa exposición é alta</string>
+    <string name="exposure_detail_high_last_update">Poderías estar infectado desde hai <b>%1$s</b> (actualizado %2$s)</string>
+    <string name="exposure_detail_high_exposure_dates_title">Historial de exposicións:</string>
+
+    <string name="exposure_detail_phone_title">Para te darmos a mellor orientación, selecciona a túa Comunidade Autónoma:</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:222]:
```suggestion
    <string name="exposure_detail_phone_title">Para dármosche a mellor orientación, selecciona a túa Comunidade Autónoma:</string>
```

**Comment by xurxodiz** [comment]
> 👋 @xurxodiz estaba a enviar suxestións de cambio, pero quizás che facilito a integración se fago un comentario xeral de actualizar "fai o favor" pola forma "por favor" ou similar dependendo do contexto. Así podes facer ti un reemplazo rápido.

Sen problema, logo á noitiña fago un commit xeral de substitución deses casos 👍

File: app/src/main/res/values-gl/strings.xml [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="exposition_block_low_description"><b>Informarémoste no caso dun posíbel contacto de risco.</b></string>
+    <string name="exposition_block_high_description"><b>Estiveches en contacto cunha persoa contaxiada de COVID-19.\nInforma no 900 112 061 (de balde)</b>.</string>
+    <string name="exposition_block_infected_description"><b>O teu diagnóstico foi enviado.\nFai o favor de aillarte durante 14 días</b>.</string>
+    <string name="exposition_block_infected_more_info">Máis información ></string>
+    <string name="exposition_block_infected_more_info_url">https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/img/20.03.24_AislamientoDomiciliario_COVID19.jpg</string>
+
+    <string name="radar_block_checked_title">&appname; activo</string>
+    <string name="radar_block_not_checked_title">&appname; inactivo</string>
+
+    <string name="radar_block_checked_description">As interaccións con móbiles próximos hanse rexistrar sempre anónimamente.</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:165]:
```suggestion
    <string name="radar_block_checked_description">As interaccións con móbiles próximos hanse rexistrar sempre anonimamente.</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 8]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="exposure_detail_what_to_do">Que debo facer?</string>
+    <string name="exposure_detail_more_info">Máis información a través <b>desta ligazón</b></string>
+    <string name="exposure_detail_acknowledgement_message">Obrigados por seguires coidándote e coidando dos demais.</string>
+
+    <string name="exposure_detail_medium">A túa exposición é media</string>
+    <string name="exposure_detail_message_medium">Poderías estar infectado</string>
+    <string name="exposure_detail_tips_subtitle_medium">Fai o favor de prestar atención a posíbeis síntomas:</string>
+    <string name="exposure_detail_clinic_indications">Segue os consellos do teu centro de saúde. Máis información <b>aquí</b></string>
+    <string name="exposure_detail_more_info_tips">Tes máis información sobre consellos, sintomas e medidas <b>aquí</b></string>
+    <string name="exposure_detail_covid_report_message"><b>Axúdanos a coidar aos demais.</b> Se te confirmaron que es un caso positivo Covid-19, axúdanos a parar a súa propagación reportándoo anónimamente</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:190]:
```suggestion
    <string name="exposure_detail_covid_report_message"><b>Axúdanos a coidar aos demais.</b> Se te confirmaron que es un caso positivo Covid-19, axúdanos a parar a súa propagación reportándoo anonimamente</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 9]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="onboarding_step_2_commitments_title_1">Anonimato</string>
+    <string name="onboarding_step_2_commitments_detail_1">A aplicación funciona sen revelar a túa identidade nin a do teu smartphone. <b>NON</b> se recolle o teu nome, email, xeolocalización, nin o teu teléfono.</string>
+    <string name="onboarding_step_2_commitments_title_2">Discreción</string>
+    <string name="onboarding_step_2_commitments_detail_2">As alertas de exposición envíanse sen indicar cando e onde se produciu a exposición.</string>
+    <string name="onboarding_step_2_commitments_title_3">Ti decides</string>
+    <string name="onboarding_step_2_commitments_detail_3">Escolle en todo momento se queres desactivar o servizo ou deixar de utilizar a App.</string>
+
+    <string name="onboarding_title_step_3">Como funciona</string>
+    <string name="onboarding_step_3_instructions_title_1">01</string>
+    <string name="onboarding_step_3_instructions_detail_1">Activa o teu Bluetooth. Iso é todo o que te pedimos.</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:112]:
```suggestion
    <string name="onboarding_step_3_instructions_detail_1">Activa o teu Bluetooth. Iso é todo o que che pedimos.</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 10]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="onboarding_step_3_instructions_title_2">02</string>
+    <string name="onboarding_step_3_instructions_detail_2">Visita a App con frecuencia para visualizares o teu grado de exposición ao coronavirus.</string>
+    <string name="onboarding_step_3_instructions_title_3">03</string>
+    <string name="onboarding_step_3_instructions_detail_3">En caso de teres estado exposto ou reportares un diagnóstico positivo segue as indicacións que te enviaremos.</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:116]:
```suggestion
    <string name="onboarding_step_3_instructions_detail_3">En caso de teres estado exposto ou reportares un diagnóstico positivo segue as indicacións que che enviaremos.</string>
```

File: app/src/main/res/values-gl/strings.xml [hunk 11]
```diff
@@ -0,0 +1,411 @@
...
+
+    <string name="network_warning_dialog_title">Sen conexión.</string>
+    <string name="network_warning_dialog_message">Fai o favor de comprobar a túa conexión e volve enviar o teu código.</string>
+    <string name="network_warning_dialog_button">Tentar de novo</string>
+
+    /********** Report Confirmation ************/
+
+    <string name="confirmation_title">Obrigados por contribuíres a parar o virus</string>
+    <string name="confirmation_subtitle"> O teu diagnóstico vainos axudar a protexer aos demais.</string>
+    <string name="confirmation_p1">Sen revelar a túa identidade en ningún momento, imos rastrexar que móbiles estiveran perto de ti nos últimos días e enviarémoslles unha alerta para a súa seguridade.\n\nFai o favor de permanecer en aillamento seguindo as recomendacións que te deron os servizos sanitarios.</string>
```
Comment (nosolosw) [inline app/src/main/res/values-gl/strings.xml:291]:
```suggestion
    <string name="confirmation_p1">Sen revelar a túa identidade en ningún momento, imos rastrexar que móbiles estiveran perto de ti nos últimos días e enviarémoslles unha alerta para a súa seguridade.\n\nPor favor, permanece en aillamento seguindo as recomendacións que che deron os servizos sanitarios.</string>
```

**Comment by xurxodiz** [comment]
@nosolosw mudanzas empuxadas 💪

File: app/src/main/res/values-gl/strings.xml [hunk 12]
```diff
@@ -42,9 +42,9 @@
...
...ase así mesmo de que esta aplicación deixará de funcionar unha vez que</b> remate a experiencia piloto.\n\n&#8226; Xa que logo, durante o uso da aplicación, <b>todas as notificacións de exposicións a posíbel contaxio da enfermidade que o usuario puider recibir corresponden a supostos simulados e, por este motivo, ao tratarse dunha experiencia piloto con datos ficticios, as suxestións de adopción de medidas preventivas e asistenciais que tras dita notificación lle facilite a aplicación non teñen outro valor do que comprobar que a aplicación é quen de facilitar suxestións dese tipo a un usuario que reciba unha notificación de exposición a posíbel contaxio.</b>\n\n&#8226; <b>Tanto no marco desta experiencia piloto con datos ficticios como no caso de que a aplicación se utilizar en situacións reais con datos reais</b>, o USUARIO fica avisado de que a utilización da Aplicación <b>NON CONSTITÚE EN NINGÚN CASO UN SERVIZO DE DIAGNÓSTICO MÉDICO, DE ATENCIÓN DE URXENCIAS OU DE PRESCRICIÓN DE TRATAMENTOS FARMACOLÓXICOS</b>, pois a utilización da Aplicación <b>non podería en ningún caso substituír a consulta presencial persoal perante un profesional médico debidamente cualificado.</b></string>
```
Comment (carlisgg) [inline app/src/main/res/values-gl/strings.xml:43]: penso que o subxuntivo en `MANIFESTAS TERES LIDO E ACEPTARES ESTAS CONDICIÓNS` está algo forzado. Coido que semella máis natural `MANIFESTAS TER LIDO E ACEPTAR ESTAS CONDICIÓNS` neste caso

File: app/src/main/res/values-gl/strings.xml [hunk 13]
```diff
@@ -0,0 +1,411 @@
...
+
+    <string name="region_info_phone_title">Teléfono COVID</string>
+    <string name="region_info_web_title">Web información COVID</string>
+
+    /********** Covid Report ************/
+
+    <string name="covid_report_title">O MEU DIAGNÓSTICO</string>
+    <string name="covid_report_subtitle">Envía o teu diagnóstico COVID positivo</string>
+    <string name="covid_report_p1">Enviando o teu diagnóstico anónimo COVID-19, estás a contribuír a deter a propagación do virus.</string>
+    <string name="covid_report_p2">Obrigados por axudarnos a coidar dos demais.</string>
```
Comment (carlisgg) [inline app/src/main/res/values-gl/strings.xml:265]: se neste caso queremos expresar agradecemento, no dicionario da RAG, obrigado non ten ese significado. Coido que, de ser o caso, habería que utilizar `Agradecidos` neste caso

File: app/src/main/res/values-gl/strings.xml [hunk 14]
```diff
@@ -0,0 +1,411 @@
...
+
+    <string name="exposure_detail_info_url">https://www.mscbs.gob.es/profesionales/saludPublica/ccayes/alertasActual/nCov-China/ciudadania.htm</string>
+
+    /********** Exposure detail low ************/
+    <string name="exposure_detail_low_title_small">RISCO BAIXO</string>
+    <string name="exposure_detail_low_title">A túa exposición é baixa</string>
+    <string name="exposure_detail_low_last_update">Sen contactos de risco aparentes. (actualizado %1$s)</string>
+    <string name="exposure_detail_low_last_update_no_data">Sen contactos de risco aparentes.</string>
+
+    <string name="exposure_detail_tips_title_low">Mantén as medidas de seguridade e distanciamiento social.</string>
```
Comment (carlisgg) [inline app/src/main/res/values-gl/strings.xml:200]: coido que sería `distanciamento`

File: app/src/main/res/values-gl/strings.xml [hunk 15]
```diff
@@ -0,0 +1,411 @@
...
+    <string name="exposure_detail_tip_mask">Usa sempre máscara cirúrxica</string>
+    <string name="exposure_detail_tip_wash_hands">Lava as mans frecuentemente</string>
+
+
+    <string name="exposure_detail_tip_distance">Mantén a distancia de seguridade de 1,5 metros</string>
+    <string name="exposure_detail_tip_clean">Limpa con regularidade as superficies que máis se tocan</string>
+    <string name="exposure_detail_tip_avoid_touch">Evita tocar os ollos, o nariz e a boca</string>
+    <string name="exposure_detail_tip_rules">Respecta as normas particulares no teu territorio sobre capacidades e actividades permitidas</string>
+    <string name="exposure_detail_tip_clinic">Ante a aparición dalgún síntoma (febre, tos, dificultade respiratoria) chama ao teu centro de saúde</string>
+    <string name="exposure_detail_tip_cough">Ao tusires ou esbirrares, tapa a boca ou o nariz e usa pano descartábeis ou a parte interior do cóbado</string>
```
Comment (carlisgg) [inline app/src/main/res/values-gl/strings.xml:211]: debería dicir `panos descartábeis`

File: app/src/main/res/values-gl/strings.xml (hunk 12 above)
Comment (xurxodiz) [reply app/src/main/res/values-gl/strings.xml:43]: É un infinitivo conxugado, non un futuro de subxuntivo, mais tes razón que sendo a mesma persoa que o verbo xa conxugado debe ir en forma invariábel. Arranxo.

File: app/src/main/res/values-gl/strings.xml (hunk 13 above)
Comment (xurxodiz) [reply app/src/main/res/values-gl/strings.xml:265]: Non é unha forma allea nin infrecuente—e de rexérmonos apenas polo dicionario da RAG, o galego ía quedar ben esmorecido—pero se preferides un `Grazas` eu trócoo.

**Comment by Yonseca** [comment]
Hi, 

Just for information. A few days ago I opened issue #33, to ask about multilanguage support. 
It seems that multilanguage support is being done using Contentful, which is a third party tool that doesn't use this strings.xml file. Similar effort has been done in PR #24, which was closed as it had no sense.

We still have no answer about how to contribute with additional languages. Another user also asked about this in  RadarCOVID/radar-covid-backend-configuration-server#4. 

Can somebody please tell us if PRs like this are useful in some way?

**Comment by xurxodiz** [comment]
I consider the answer given in [radar-covid-backend-configuration-server#5](https://github.com/RadarCOVID/radar-covid-backend-configuration-server/issues/5) satisfactory enough, so I'll close this.

Still, I must say that I'm not happy that proper state language support is done almost as an afterthought, coming even later than English, and I hope that the options offered are not limited to coofficiality—such that others like Asturian also find their way into the app.

**Comment by enolp** [comment]
Xurxodiz, check my response to https://github.com/RadarCOVID/radar-covid-backend-configuration-server/issues/5#issuecomment-694236338 because I don't want to repeat my words in every place.

It is sad but Gobierno de España still thinks that if a language is not (co-)official, it does not exist even if there is some law which protect it AND it is PUBLISHED in Boletín Oficial del Estado.

On the other hand, Asturies and Aragon are guilty by not making its own languages co-official in 42 years of the Autonomous Community system.

**Pull request closed without merging**